Formula/go.rb

```diff
@@ -3,27 +3,16 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.6.3.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.6.3.src.tar.gz"
-    version "1.6.3"
-    sha256 "6326aeed5f86cf18f16d6dc831405614f855e2d416a91fd3fdc334f772345b00"
-
-    # 1.6.3 does not build on macOS Sierra. Users should use devel instead
-    # until 1.7 is stable (due soon).
-    depends_on MaximumMacOSRequirement => :el_capitan
-
-    # Should use the last stable binary release to bootstrap.
-    resource "gobootstrap" do
-      url "https://storage.googleapis.com/golang/go1.6.2.darwin-amd64.tar.gz"
-      version "1.6.2"
-      sha256 "6ebbafcac53bbbf8c4105fa84b63cca3d6ce04370f5a04ac2ac065782397fc26"
-    end
+    url "https://storage.googleapis.com/golang/go1.7.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.7.src.tar.gz"
+    version "1.7"
+    sha256 "72680c16ba0891fcf2ccf46d0f809e4ecf47bbf889f5d884ccb54c5e9a17e1c0"
 
-    go_version = "1.6"
+    go_version = "1.7"
```

Comment:
would prefer go_version = version.to_s.split(".")[0..1].join(".") if the goal is to use "1.7" for "1.7", "1.7.1", "1.7.2", ...